### PR TITLE
Update contributing.rst

### PR DESCRIPTION
**Pull request by aphelionz**

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -47,18 +47,18 @@ The Hydroshare team does our best to respond to pull requests within three busin
 
 Starting on Sept 1, 2017, CUAHSI will assume the role of the "benevolent dictator" of this production Hydroshare code base. After that point, pull requests into develop (and ultimately master) will only be merged by CUAHSI staff or other appointed staff from partner institutions. Other users can still:
```
Comment (Castronova) [inline docs/contributing.rst]: role of the "benevolent dictator" of this --> role of the "benevolent dictator" of the

File: docs/contributing.rst [hunk 2]
```diff
@@ -47,18 +47,18 @@ The Hydroshare team does our best to respond to pull requests within three busin
...
 
-#) Create issues pull requests
+#) Create issues and/or pull requests
 #) Request reviews from anybody.
```
Comment (Castronova) [inline docs/contributing.rst]: 3. Participate in code review

File: docs/contributing.rst [hunk 3]
```diff
@@ -47,18 +47,18 @@ The Hydroshare team does our best to respond to pull requests within three busin
...
 
-Pull requests will not be merged until at least one code review is accepted. We may suggest some changes or improvements or alternatives. Some things that will increase the chance that your pull request is accepted:
+Pull requests will not be merged until at least one code review is accepted. Additionally, a likely-non-coder stakeholder must review the work to make sure that their issue is resolved or that it doesn't cause any additional technical or philosophical problems.
```
Comment (Castronova) [inline docs/contributing.rst]: a likely-non-coder stakeholder --> a stakeholder outside of the core development team

File: docs/contributing.rst [hunk 4]
```diff
@@ -47,18 +47,18 @@ The Hydroshare team does our best to respond to pull requests within three busin
...
+
+We may suggest some changes or improvements or alternatives. Some things that will increase the chance that your pull request is accepted:
```
Comment (Castronova) [inline docs/contributing.rst]: We may suggest some changes or improvements or alternatives.  --> This review process may result in suggested changes, improvements, or alternative designs that will need to be addressed.

Comment (Castronova) [inline docs/contributing.rst]: Some things that will increase the chance that your pull request is accepted: --> Below is a list of best practices that will help expedite the process of accepting developer code into the master branch:

File: docs/contributing.rst [hunk 5]
```diff
@@ -47,18 +47,18 @@ The Hydroshare team does our best to respond to pull requests within three busin
...
 
 - A reference to the corresponding issue in the description of the pull request.
```
Comment (Castronova) [inline docs/contributing.rst]: referencing specific issue(s) that have been addressed in the pull request

File: docs/contributing.rst [hunk 6]
```diff
@@ -47,18 +47,18 @@ The Hydroshare team does our best to respond to pull requests within three busin
...
 - Create an issue to pair your pull request with
```
Comment (Castronova) [inline docs/contributing.rst]: Pairing your pull request with a single, well documented, Github issue

File: docs/contributing.rst [hunk 7]
```diff
@@ -47,18 +47,18 @@ The Hydroshare team does our best to respond to pull requests within three busin
...
 - Write tests
```
Comment (Castronova) [inline docs/contributing.rst]: Writing unit tests

File: docs/contributing.rst [hunk 8]
```diff
@@ -47,18 +47,18 @@ The Hydroshare team does our best to respond to pull requests within three busin
...
 - Follow the `PEP8 style guide`_
```
Comment (Castronova) [inline docs/contributing.rst]: Following

File: docs/contributing.rst [hunk 9]
```diff
@@ -47,18 +47,18 @@ The Hydroshare team does our best to respond to pull requests within three busin
...
 - Write a `good commit message`_
```
Comment (Castronova) [inline docs/contributing.rst]: Writing clear and concise `commit` messages to help code reviewers

File: docs/contributing.rst (hunk 8 above)
Comment (aphelionz) [inline docs/contributing.rst]: hmm?

Comment (Castronova) [inline docs/contributing.rst]: Following the `PEP8 style guide`

**Pull request merged**